internal/conf/server.go

```diff
@@ -11,7 +11,7 @@ import (
 // "raw" configuration.
 type ConfigurationSource interface {
 	// Write updates the configuration. The Deployment field is ignored.
-	Write(ctx context.Context, data conftypes.RawUnified, lastID int32) error
+	Write(ctx context.Context, data conftypes.RawUnified, lastID int32, authorUserID *int32) error
```

Comment:
I'd personally do this and treat `0` as no-author, just to avoid some potential nil panics etc.

```suggestion
	Write(ctx context.Context, data conftypes.RawUnified, lastID int32, authorUserID int32) error
```